### PR TITLE
Science blog numbering and some corrections

### PR DESCRIPTION
**Pull request by dsarkar**
- Science blogs are sequentially numbered
- sitemap updated, one blog article was actually missing
- more concise wording of fig captions in science blog 4

Internal Tracking ID: [EXPOSUREAPP-14561](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-14561)

### CODE REVIEW

**Review by Ein-Tim — APPROVED** [review]
Had a quick look, not checked for typos: Looks good to me!

**Comment by dsarkar** [comment]
@Ein-Tim thanks, there will be another commit with some corrections in the fig captions of science blog 4

File: science/2021-10-15-science-blog-4/index_de.md [hunk 1]
```diff
@@ -398,27 +398,27 @@ Nach 48 Stunden nach einer Warnung waren insgesamt 25.518 Antigen-Schnelltests (
...
+<figcaption aria-hidden="true">Abb. 15: Zeit zwischen Risikobegegnung und Testregistrierung (in Tagen vor Testregistrierung) (PCR).</figcaption>
 
 Die tagesgenaue Verteilung des Verzugs zwischen Risikobegegnung und Registrierung eines Antigen-Schnelltests ist in der folgenden Abbildung dargestellt. Im Mittel wurde der Test 7,4 Tage nach der Risikobegegnung registriert (Median: 7 Tage).
 
-<img src="./ppa_delay_er_rat_plot.png" class="figure-img img-fluid" alt="Zeit zwischen Risikobegegnung und Testregistrierung (nach Tagen vor Testregistrierung) (Antigen-Schnelltest)."/>
-<figcaption aria-hidden="true">Abb. 16: Zeit zwischen Risikobegegnung und Testregistrierung (nach Tagen vor Testregistrierung) (Antigen-Schnelltest).</figcaption>
+<img src="./ppa_delay_er_rat_plot.png" class="figure-img img-fluid" alt=" Zeit zwischen Risikobegegnung und Testregistrierung (in Tagen vor Testregistrierung) (Antigen-Schnelltest)."/>
+<figcaption aria-hidden="true">Abb. 16:  Zeit zwischen Risikobegegnung und Testregistrierung (in Tagen vor Testregistrierung) (Antigen-Schnelltest).</figcaption>
```
Comment (Ein-Tim) [inline science/2021-10-15-science-blog-4/index_de.md:407]: Remove unnecessary white space.
```suggestion
<img src="./ppa_delay_er_rat_plot.png" class="figure-img img-fluid" alt="Zeit zwischen Risikobegegnung und Testregistrierung (in Tagen vor Testregistrierung) (Antigen-Schnelltest)."/>
<figcaption aria-hidden="true">Abb. 16: Zeit zwischen Risikobegegnung und Testregistrierung (in Tagen vor Testregistrierung) (Antigen-Schnelltest).</figcaption>
```

**Comment by Ein-Tim** [comment]
The white space changes to the English file `science/2021-10-15-science-blog-4/index.md` differ from the one done to `science/2021-10-15-science-blog-4/index_de.md`. I only commented on the lines where a double whitespace used.

**Review by mtb77 — APPROVED** [review]
